architecture-compose/src/androidLibMain/kotlin/com/splendo/kaluga/architecture/compose/navigation/RouteNavigator.kt

```diff
@@ -0,0 +1,85 @@
+package com.splendo.kaluga.architecture.compose.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import com.splendo.kaluga.architecture.navigation.NavigationAction
+import com.splendo.kaluga.architecture.navigation.Navigator
+import kotlin.reflect.KClass
+
+const val BACK_ROUTE = "back"
```

Comment:
Can it be names collision if I create my own NavigationAction with name 'back'?

Reply:
updated constant so it won't be so easy to clash